packages/web-app-files/src/helpers/share/link.ts

```diff
@@ -28,7 +28,7 @@ export const createQuicklink = async (args: CreateQuicklink): Promise<Share> =>
   const canEdit = store.state.user.capabilities.files_sharing?.public?.can_edit || false
   const canContribute = store.state.user.capabilities.files_sharing?.public?.can_contribute || false
   const alias = store.state.user.capabilities.files_sharing?.public?.alias
-  const roleName = !canCreatePublicLink ? 'none' : capabilitiesRoleName || 'viewer'
+  const roleName = !canCreatePublicLink ? 'internal' : capabilitiesRoleName || 'viewer'
```

Comment:
Could you please use the link role constants here as well and reference their `name` props? Please always use the `folder` constants for referencing. The names are identical for file and folder role constants, but we always use the `folder` version when referencing role names.